test/unittest/StatisticsBackend/BackendDumpTests.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <list>
+#include <fstream>
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <StatisticsBackend.hpp>
+#include <StatisticsBackendData.hpp>
+
+#include <database/database.hpp>
+
+using namespace eprosima::statistics_backend;
+using namespace eprosima::statistics_backend::database;
+
+void load_file(
+        std::string filename,
+        DatabaseDump& dump)
+{
+    constexpr const char* DESCRIPTION_TAG = "description";
+
+    // Check if the file exists
+    std::ifstream file(filename);
+    if (!file.good())
+    {
+        FAIL() << "File " + filename + " does not exist";
+    }
+
+    // Get the json
+    file >> dump;
+
+    // Erase the description tag if existing
+    if (dump.contains(DESCRIPTION_TAG))
+    {
+        dump.erase(DESCRIPTION_TAG);
+    }
+}
+
+TEST(backend_dump_tests, reset)
+{
+    constexpr const char* DUMP_FILE = "../Resources/simple_dump.json";
```

Comment:
When StatisticsBackend::is_active() [11667] is merged, this, and other file strings can be deleted and use instead the declared on `databaseUtils.cpp`, since is used for several tests.